src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+public class Main {
+    public static void main(String[] args) {
+        final int numberElementsArray = 6;
```

Comment:
is `numberElementsArray` a constant?
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names